lib/CustomFields/pages/EditCustomFieldsRecord/EditCustomFieldsRecord.js

```diff
@@ -225,6 +225,7 @@ const EditCustomFieldsRecord = ({
       initializeFields();
       onComponentLoad?.();
     }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
It is fine to do this, but please add a comment explaining why it is necessary. You have it nicely documented in this PR, but users browsing through the source won't see these comments and will have to go spelunking through git history to figure out what's going on here.